package.json

```diff
@@ -105,6 +105,7 @@
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-standard": "^5.0.0",
+    "eslint-plugin-testing-library": "^6.3.0",
```

Comment:
Where is this used?

Reply:
I belive it's being used to lint testing-library

Reply:
Yes, it's React Testing Library (RTL) eslint suggested linter dependency.